src/components/Sphere/index.tsx

```diff
@@ -1,12 +1,23 @@
-import { css } from '@emotion/react'
+import { Theme, css } from '@emotion/react'
 import styled from '@emotion/styled'
 import PropTypes from 'prop-types'
-import React from 'react'
+import React, { VoidFunctionComponent } from 'react'
+import { Color } from '../../theme/colors'
 import Box from '../Box'
 
-const bordersStyles = ({ size, bgColors, theme }) => {
+const bordersStyles = ({
+  size,
+  bgColors,
+  theme,
+}: {
+  size: number
+  bgColors: string[]
```

Comment:
```suggestion
  bgColors: (string | Color)[]
```

Reply:
This would break the propTypes below, we have no way to specify that a prop need to be a Color for now :)